opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/consent/AccountAccessType.java

```diff
@@ -0,0 +1,27 @@
+package de.adorsys.opba.protocol.xs2a.service.xs2a.dto.consent;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum AccountAccessType {
+    ALL_ACCOUNTS("ALL_ACCOUNTS"),
+    ALL_ACCOUNTS_WITH_BALANCES("ALL_ACCOUNTS_WITH_BALANCES");
+
+    private static Map<String, AccountAccessType> container = new HashMap<>();
```

Comment:
Why do you need HashMap here?
This should work for your use case:
```
@Getter
@RequiredArgsConstructor
public enum AccountAccessType {
    ALL_ACCOUNTS("ALL_ACCOUNTS"),
    ALL_ACCOUNTS_WITH_BALANCES("ALL_ACCOUNTS_WITH_BALANCES");

   private final String apiName;
}
```

Reply:
HashMap was removed